services/Logic/workflows/alerts.yaml

```diff
@@ -4,8 +4,8 @@
   verified: false
   visible: true
   tags:
-  - auto-generated
-  - agc-62008
+    - auto-generated
```

Comment:
I think the format changes need to reverted. @JoeyBarnes can you check/approve the changes to the yaml? Suggest to check changes to format, GUID, Tags for the added alert.